frontend/src/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { BrowserRouter } from "react-router-dom";
+import AppRoutes from "./route.tsx";
+
+const rootElement = document.getElementById("root");
+
+if (rootElement) {
+  const root = ReactDOM.createRoot(rootElement);
+  root.render(
+    <BrowserRouter
```

Comment:
[Question] Using experimental or future features might makes unstable the code, why we need this?